README.md

```diff
@@ -1,2 +1,21 @@
 # puzzle-solver
 A bachelor thesis project at Chalmers University of Technology
+
+Git Good, 
+A guide for nice git-behaviour
+Branch naming convention: 
+- All lower case and spaces using -
+- feature/{issue name}
+  - eg. /feature/sudoku-parser
```

Comment:
This looks strange, I do not understand what is happening. Will this break the code?